diff -urNp sed-4.2.1-orig/sed/sed.c sed-4.2.1/sed/sed.c
--- sed-4.2.1-orig/sed/sed.c	2011-07-28 10:20:07.555588634 +0200
+++ sed-4.2.1/sed/sed.c	2011-07-28 11:04:55.315831155 +0200
@@ -135,12 +135,19 @@ Usage: %s [OPTION]... {script-only-if-no
                  add the contents of script-file to the commands to be executed\n"));
 #ifdef ENABLE_FOLLOW_SYMLINKS
   fprintf(out, _("  --follow-symlinks\n\
-                 follow symlinks when processing in place\n"));
+                 follow symlinks when processing in place; hard links\n\
+                 will still be broken.\n"));
 #endif
   fprintf(out, _("  -i[SUFFIX], --in-place[=SUFFIX]\n\
-                 edit files in place (makes backup if extension supplied)\n"));
+                 edit files in place (makes backup if extension supplied).\n\
+                 The default operation mode is to break symbolic and hard links.\n\
+                 This can be changed with --follow-symlinks and --copy.\n"));
   fprintf(out, _("  -c, --copy\n\
-                 use copy instead of rename when shuffling files in -i mode\n"));
+                 use copy instead of rename when shuffling files in -i mode.\n\
+                 While this will avoid breaking links (symbolic or hard), the\n\
+                 resulting editing operation is not atomic.  This is rarely\n\
+                 the desired mode; --follow-symlinks is usually enough, and\n\
+                 it is both faster and more secure.\n"));
 #if defined(WIN32) || defined(_WIN32) || defined(__CYGWIN__) || defined(MSDOS) || defined(__EMX__)
   fprintf(out, _("  -b, --binary\n\
                  open files in binary mode (CR+LFs are not processed specially)\n"));
